src/map/packets_struct.hpp

```diff
@@ -5252,6 +5252,28 @@ struct PACKET_CZ_REQ_TAKEOFF_EQUIP_ALL {
 DEFINE_PACKET_HEADER(CZ_REQ_TAKEOFF_EQUIP_ALL, 0x0bad);
 #endif  // PACKETVER_MAIN_NUM >= 20210818 || PACKETVER_RE_NUM >= 20211103 || PACKETVER_ZERO_NUM >= 20210818
 
```

Comment:
Please move this to packets.hpp
This is the Hercules file and should not be modified.